scripts/assets/accessibility_label_exemptions.textproto

```diff
@@ -1,4 +1,7 @@
+exempted_activity: "app/src/main/java/org/oppia/android/app/devoptions/markchapterscompleted/testing/MarkChaptersCompletedTestActivity"
+exempted_activity: "app/src/main/java/org/oppia/android/app/devoptions/markstoriescompleted/testing/MarkStoriesCompletedTestActivity"
```

Comment:
Shouldn't these be added in earlier PRs, or were they missed?

Reply:
They were missed actually.